lib/cog/command/command_resolver.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Cog.Command.CommandResolver do
+
+  alias Cog.Repo
+  alias Cog.Queries.Command
+  alias Cog.Queries.Alias
+  alias Piper.Command.SemanticError
+
+  def find_bundle(name) when is_binary(name) do
+    if String.contains?(name, ":") do
+      :identity
+    else
+      case Repo.all(Command.bundle_for(name)) do
```

Comment:
Maybe we do something like when an unqualified command name is not authoritative (i.e., "You typed `foo`, but we don't know if you meant `foo:foo` or `bar:foo`")

Perhaps we need a way to unambiguously note that something is an alias? (`alias:foo`, though that prevents there ever being a bundle named `alias`...)